src/libraries/System.Runtime.InteropServices.RuntimeInformation/tests/DescriptionNameTests.cs

```diff
@@ -180,6 +180,29 @@ public void VerifyRuntimeNameOnNetCoreApp()
             Assert.Same(RuntimeInformation.FrameworkDescription, RuntimeInformation.FrameworkDescription);
         }
 
+        [Fact]
+        public void VerifyFrameworkDescriptionContainsCorrectVersion()
+        {
+            var frameworkDescription = RuntimeInformation.FrameworkDescription;
+            var version = frameworkDescription.Substring(".NET".Length).Trim(); // remove ".NET" prefix
+
+            if (string.IsNullOrEmpty(version))
+                return;
```

Comment:
Shouldn't this be an assert instead?

Reply:
The code explicitly handles the case where there is no version information so the test should too:
https://github.com/dotnet/runtime/blob/08903c00860939af0d291eb3f3c037a18d9820f6/src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/RuntimeInformation.cs#L29